Gaijinizer patch/Map050.txt

```diff
@@ -955,10 +959,10 @@ is no corpse.#MTLed
 遺体はおろか、食い散らかされた血痕や肉片すらない事が。
 > CONTEXT: Map050/events/57/pages/1/46/Dialogue/kazuya
 > CONTEXT: Map188/events/57/pages/1/46/Dialogue/kazuya
-So I asked Yuuma a little earlier.
-It seems that Yuuma was also strange.
-Don't let the dead, eat away bloodstains and
-shreds of meat.#MTLed
+So I asked Yuuma about it earlier. Seems he also
+thought it was strange. Forget about the bodies,
+there aren't even any bloodstains of eaten bodies,
+no hunks of flesh lying around.#Ron
```

Comment:
Flow:
>_**So, I asked Yuuma about it earlier... 
Seems he also thought it was strange. 
Forgot about the bodies, there aren't even
any hunks of flesh or bloodstains around.**_

(Mentioning bodies twice in the same sentence seems redundant, and it mentally makes sense to go from the most intense thing to the least).